src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -153,30 +174,6 @@ class IntervalExpressionSegment(BaseSegment):
     )
 
 
-@bigquery_dialect.segment(replace=True)
```

Comment:
Removing this in favor of overriding `SelectClauseModifierSegment`, as explained above.

Reply:
yes, more code re-use from the base dialect :)